source/adapters/native_cpu/kernel.cpp

```diff
@@ -171,6 +172,10 @@ urKernelGetSubGroupInfo(ur_kernel_handle_t hKernel, ur_device_handle_t hDevice,
     // todo: set proper values
     return ReturnValue(0);
   }
+  case UR_KERNEL_SUB_GROUP_INFO_FORCE_UINT32: {
```

Comment:
this is not really a real value.
`default: {
} break;`
?

Reply:
This should be an unreachable case so I agree with the second suggestion.

Reply:
done

Reply:
could be useful to have something like unreachable in common, but otherwise lgtm.

Reply:
Would `std::abort()` suffice?